src/evict/evict_lru.c

```diff
@@ -1570,7 +1773,7 @@ __evict_get_ref(
 	server_only = is_server && !WT_EVICT_HAS_WORKERS(session);
 	urgent_ok = (!is_app && !is_server) ||
 	    !WT_EVICT_HAS_WORKERS(session) ||
-	    (is_app && __wt_cache_aggressive(session));
+	    __wt_cache_aggressive(session);
```

Comment:
Same with this removal/change.  This line was changed Oct23 and seems unrelated to your work.